tools/serverpod_cli/lib/src/analyzer/entities/validation/keywords.dart

```diff
@@ -13,12 +13,14 @@ class Keyword {
   static const String unique = 'unique';
   static const String parent = 'parent';
   static const String relation = 'relation';
+  static const String onUpdate = 'onUpdate';
+  static const String onDelete = 'onDelete';
   static const String api = 'api';
   static const String database = 'database';
   static const String optional = 'optional';
   static const String scope = 'scope';
   static const String persist = 'persist';
 
   /// Special keyword to allow keys to be any string.
-  static const String any = 'any';
+  static const String any = '#any';
```

Comment:
Question: This is a breaking change right or has this not been used before?

Reply:
No breaking change, I made this change to make sure we have a key that would never match with a user defined key. If a user previously added a field in a class called "any" (perhaps not common, but still) we would think it was our special keyword `any` which in this context has a special meaning while parsing / validating. 

By changing this to `#any` this can never happen as the user is not allowed to name a field `#any`.